src/box/box.cc

```diff
@@ -2925,6 +2925,13 @@ box_feedback_report_crash(struct crash_info *cinfo)
 void
 box_crash_callback(struct crash_info *cinfo)
 {
+	if (cinfo->signo == SIGBUS &&
+	    flightrec_is_mmapped_address(cinfo->siaddr)) {
+		fprintf(stderr, "error accessing flightrec file\n");
+		fflush(stderr);
+		_exit(EXIT_FAILURE);
```

Comment:
The callback is invoked after dumping the stack trace to stderr. I think it should be called before, because a stack trace looks pretty scary to users. AFAIR we agreed to *replace* the default crash callback instead of adding an extra handler and add make the default handler (the one that dumps the stack trace) available outside crash.c so that it can be reused in box.cc.

Reply:
Issue with reporting crash info on flightrec SIGBUS is fixed.

As to error message - leave "as is" as discussed f2f.